### PR TITLE
Args qol updates

### PR DESCRIPTION
**Pull request by marsninja**
@ypkang Can you review this. Im thinking you can start reviewing my PRs as a learning tool. We can have calls to talk through them, for fun, learning, and some feedback for me :-).

### CODE REVIEW

**Comment by ypkang** [comment]
> @ypkang Can you review this. Im thinking you can start reviewing my PRs as a learning tool. We can have calls to talk through them, for fun, learning, and some feedback for me :-).

Love it! Looking through this one right now.

File: jaseci_core/jaseci/api/alias_api.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class alias_api():
     def __init__(self):
         self.alias_map = {}
 
-    @interface.private_api()
+    @interface.private_api(cli_args=['name'])
```
Comment (ypkang) [inline jaseci_core/jaseci/api/alias_api.py:26]: So this means that `alias register` will have the default arguments `name`, correct? 

```
alias register NAME
```

File: jaseci_core/jaseci/api/architype_api.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class architype_api():
     a Jaseci instance is aware of.
     """
 
-    @interface.private_api()
+    @interface.private_api(cli_args=['code'])
```
Comment (ypkang) [inline jaseci_core/jaseci/api/architype_api.py:23]: Can I have multiple `cli_args`? I'd assume no because then their position would matter, right?

**Review by ypkang — APPROVED** [review]

**Comment by ypkang** [comment]
This looks good! I just have a couple of clarifying questions to help me understand better how this works. Left them as comments inline.

File: jaseci_core/jaseci/api/alias_api.py (hunk 1 above)
Comment (marsninja) [reply jaseci_core/jaseci/api/alias_api.py:26]: Correct! No need for `alias register -name NAME -value VALUE` you can just do `alias register NAME -value VALUE`

File: jaseci_core/jaseci/api/architype_api.py (hunk 2 above)
Comment (marsninja) [reply jaseci_core/jaseci/api/architype_api.py:23]: You can have multiple, just would need to keep in mind whatever is in the list will have to be the first args to the given command in that order then you'd use the `-param {value}` in any order for other params.

**Pull request merged**